script/dump_templates

```diff
@@ -104,7 +104,7 @@ JobTemplates e.g. to load them on another instance.
 
 BEGIN {
     use FindBin qw($Bin);
```

Comment:
I see down you swap `$Bin` with `$RealBin`, is the import still needed?

Reply:
correct, not needed. Updated and harmonized clone_job.pl in the same way.